ecs-cli/modules/aws/clients/ecs/client.go

```diff
@@ -133,15 +133,21 @@ func (client *ecsClient) DeleteService(serviceName string) error {
 	return nil
 }
 
-func (client *ecsClient) CreateService(serviceName, taskDefName string, deploymentConfig *ecs.DeploymentConfiguration) error {
-	_, err := client.client.CreateService(&ecs.CreateServiceInput{
+func (client *ecsClient) CreateService(serviceName, taskDefName string, loadBalancer *ecs.LoadBalancer, role string, deploymentConfig *ecs.DeploymentConfiguration) error {
+	createServiceInput := &ecs.CreateServiceInput{
 		DesiredCount:            aws.Int64(0),            // Required
 		ServiceName:             aws.String(serviceName), // Required
 		TaskDefinition:          aws.String(taskDefName), // Required
 		Cluster:                 aws.String(client.params.Cluster),
 		DeploymentConfiguration: deploymentConfig,
-	})
-	if err != nil {
+	}
+
+	if *loadBalancer.TargetGroupArn != "" || *loadBalancer.LoadBalancerName != "" {
```

Comment:
* I understand that for the current use case, the loadBalancer will always be set and will never be nil, but this `CreateServce` method can be reused in the future, hence I would still recommend adding a nil pointer check here.

* Can you change `*loadBalancer.TargetGroupArn` to use aws.StringValue (`aws.StringValue(loadBalancer.TargetGroupArn)` and `aws.StringValue(loadBalancer.LoadBalancerName)`)